workers/common/src/main/kotlin/common/OrtRunService.kt

```diff
@@ -276,6 +281,39 @@ class OrtRunService(
         getScannerJobForOrtRun(ortRunId)?.let { scannerRunRepository.getByJobId(it.id) }
     }
 
+    /**
+     * Loads the results of the previous worker steps and generates an [OrtResult] from them. In addition, it
```

Comment:
Nit: In contrast to Java, the convention for Kotlin comments is to use an imperative style, i.e. "Load the results...". I am personally also more used to the Java conventions, but throughout the codebase the Kotlin style is used.